jina/jaml/parsers/executor/legacy.py

```diff
@@ -98,6 +99,16 @@ def parse(self, cls: Type['BaseExecutor'], data: Dict) -> 'BaseExecutor':
         obj.is_updated = False
         return obj
 
+    @staticmethod
+    def is_tail_or_head(data: Dict) -> bool:
+        """Based on name, compute if this is a tail/head Pea or a main Pea
+
+        :param data: the data for the parser
+        :return: boolean of whether it is tail/head or main
```

Comment:
that description is slightly misleading? I assume this is True if it is head/tail . 

So I'd write :return: boolean True if it is tail/head., False otherwise

Reply:
adjusted, thanks